draft-liu-teas-transport-network-slice-yang.txt

```diff
@@ -0,0 +1,1568 @@
+       |        |     +--rw bound?               uint64
+       |        +--rw security*                 identityref
+       |        +--rw isolation?                identityref
+       |        +--rw max-occupancy-level?      uint8
+       |        +--rw mtu?                      uint16
+       |        +--rw steering-constraints
+       |        |  +--rw path-constraints
+       |        |  +--rw service-function
+       |        |  +--rw disjointness?
+       |        |  |       te-types:te-path-disjointness
+       |        |  +--rw service-info?       string
+       |        +--rw optimization-criterion?   identityref
+       |        +--rw delay-tolerance?          boolean
+       |        +--rw periodicity*              uint64
+       |        +--rw resize-requirement?       identityref
+       |        +--rw service-info?             string
+
+
+
+Liu, et al.              Expires 13 August 2023                [Page 10]
+
+Internet-Draft      Network Slice Topology Data Model      February 2023
+
+
+       +--rw optimization-criterion?         identityref
+       +--rw delay-tolerance?                boolean
+       +--rw periodicity*                    uint64
+       +--rw resize-requirement?             identityref
+       +--rw service-info?                   string
+
+             Figure 3: Tree diagram for network slice topology
+
+6.  YANG Modules
+
+   <CODE BEGINS> file "ietf-ns-topo@2023-01-31.yang"
+      module ietf-ns-topo {
+        yang-version 1.1;
+        namespace
+          "urn:ietf:params:xml:ns:yang:ietf-ns-topo";
+        prefix "ns-topo";
+
+        import ietf-network {
+          prefix "nw";
+          reference "RFC 8345: A YANG Data Model for Network Topologies";
+        }
+        import ietf-network-topology {
+          prefix "nt";
+          reference "RFC 8345: A YANG Data Model for Network Topologies";
+        }
+
+        import ietf-te-types {
+          prefix "te-types";
+          reference
+            "RFC 8776: Traffic Engineering Common YANG Types";
+        }
+
+        import ietf-network-slice-service {
+          prefix "ietf-nss";
+          reference
+            "draft-ietf-teas-ietf-network-slice-nbi-yang-00:
+             IETF Network Slice Service YANG Model";
+        }
+
+        organization
+          "IETF CCAMP Working Group";
+        contact
+          "WG Web: <http://tools.ietf.org/wg/ccamp/>
+           WG List: <mailto:ccamp@ietf.org>
+
+           Editor: Xufeng Liu
+                   <mailto:xufeng.liu.ietf@gmail.com>
+
+
+
+
+Liu, et al.              Expires 13 August 2023                [Page 11]
+
+Internet-Draft      Network Slice Topology Data Model      February 2023
+
+
+           Editor: Italo Busi
+                   <mailto:italo.busi@huawei.com>
+
+           Editor: Aihua Guo
+                   <mailto:aihuaguo.ietf@gmail.com>
+
+           Editor: Sergio Belotti
+                   <mailto:sergio.belotti@nokia.com>";
+
```

Comment:
Could I be added as editor of the YANG model? I didn't have yet the chance of being listed as editor of any YANG module, so it is just personal goal :-)

Reply:
Absolutely!